src/main/java/com/github/sardine/ant/SardineTask.java

```diff
@@ -82,7 +91,17 @@ private void addCommand(Command command) {
 	@Override
 	public void execute() throws BuildException {
 		try {
-			fSardine = SardineFactory.begin(fUsername, fPassword);
+			if ( fDomain==null && fWorkstation==null ) {
```

Comment:
minor formatting, no spaces between the parens and spaces between the conditional. `if (fDomain == null && fWorkstation == null)`